src/components/screens/votingQueue/styles.css

```diff
@@ -0,0 +1,70 @@
+@import '../../../app/mixins.css';
+
+.modal {
+  width: 45%;
+  min-width: 638px;
+  background: var(--color-white);
+  border-radius: var(--border-radius-box);
+  position: absolute;
+  top: 0;
+  left: 216px;
+}
+
+.toggleIcon {
+  padding-left: 14px;
+  box-sizing: border-box;
+
+  & :hover {
+    cursor: pointer;
+  }
+}
+
+.toggleIcon.notHeader {
+  border-bottom: 1px solid #dfe6f2;
```

Comment:
This color is used on other border bottoms. It can be a good time to create a global variable containing it.

Reply:
Done!